.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Operating System:
+- Node Version:
+- NPM Version:
+- Rollup Version:
+- ${package} Version:
+
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+### Code
+
+```js
+// rollup.config.js
+
+// If your Rollup config is meaty 🍖 and over 20 lines long, please provide a
+// URL to a repo for your app or link to a gist https://gist.github.com
+```
+
+```js
+// additional code. remove this block if you don't need it
+```
+
+### How Do We Reproduce?
```

Comment:
Only necessary if we do not have a REPL link.

Reply:
I've moved this to the top so it's the priority, and added commented out notes for how to provide a minimal reproduction, which are very similar to the template that exists on master. It'll give users a choice for how to provide a repro, and a space for elaborating if needs be.